src/lib/react-server-route.js

```diff
@@ -30,12 +30,12 @@ function routeFileContent(locales) {
     function generateHTML(renderProps) {
       var messages = getMessages(locale);
       var favicon = "/assets/images/favicon.d0608f227db61f2852a32087e614911c.png";
-      var twitterImage = "/assets/images/EOY_Twitter_v8_EN.d1bb5d2a5ce35859d038df852d9e6a0a811beaac.png";
-      var facebookImage = "/assets/images/EOY_facebook_v1.a152496406bad899d1a920f6d6b9f507.png";
```

Comment:
Do these images still exist in the repo and if so, can we delete them or is there anywhere we're using them still?